tests/TestCompose_nutils-of.Ubuntu1804.home/Dockerfile.nutils

```diff
@@ -16,8 +16,10 @@ ARG branch=develop
 # Builds the precice python bindings for python3
 RUN pip3 install --user https://github.com/precice/python-bindings/archive/$branch.zip
 
+ARG version=v6.0
+
 WORKDIR /home/precice/
-RUN git clone https://github.com/nutils/nutils.git && \
+RUN git clone --branch $version https://github.com/evalf/nutils.git && \
```

Comment:
It turns out that we would install nutils by cloning the master branch of the nutils repository. Since the start of v7.0 development, this causes the test to fail as described in #219. Therefore, we now clone a specific branch to consistently use a working version.

Reply:
Note also that the nutils repository migrated to a new location, hence the URL change.